packages/nvidia_nat_vanna/src/nat/plugins/vanna/vanna_utils.py

```
@@ -0,0 +1,845 @@
+            output_field = collection_name
+
+        record_list = []
+        try:
+            records = await self.async_milvus_client.query(
+                collection_name=collection_name,
+                output_fields=[output_field],
+                limit=self.milvus_search_limit,
+            )
+            for record in records:
+                record_list.append(record[output_field])
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)